nilearn/conftest.py

```diff
@@ -89,7 +94,114 @@ def close_all():
 )
 
 
-def _mni_3d_img(affine=MNI_AFFINE):
+def _affine_mni():
```

Comment:
Does anything prevent from always using this even inside conftest? What is the benefit of still having the global?

Reply:
yeah I was thinking that the global was progressively becoming useless. 

will try to simplify.

Reply:
But isn't that a breaking change ? I'd rather have deprecation cycles...

Reply:
looking through all the files changed, all those constants only appear in tests so I don't think deprecation is necessary there.

are we actually storing anything in conftest.py that is not just for testing? if so we should probably move it somewhere else